server/function_collection.cc

```diff
@@ -214,16 +234,29 @@ initRoutes(RouteManager & manager)
     auto mapDesc = std::make_shared<MapDescription<Utf8String, ExpressionValue> >
         (getExpressionValueDescriptionNoTimestamp());
 
+    const auto inputDefStr = "Object with input values. "
+                             "Must be defined either as a query string "
+                             "parameter or the json body.";
+    const auto keepValuesDefStr = "Keep only these values for the output. "
+                                  "Must be defined either as a query string "
+                                  "parameter or the json body.";
+
     addRouteAsync(*manager.valueNode, "/application", { "GET" },
                   "Apply a function to a given set of input values and return the output",
                   //"Output of all values or those selected in the keepValues parameter",
                   &FunctionCollection::applyFunction,
                   manager.getCollection,
                   getFunction,
-                  RestParamJson<std::map<Utf8String, ExpressionValue> >("input", "Object with input values", JsonStrCodec<MapType>(mapDesc)),
-                  RestParamJsonDefault<std::vector<Utf8String> >
-                  ("keepValues", "Keep only these values for the output", {}),
-                  PassConnectionId());
```

Comment:
I wonder to what extent this mechanim should actually be moved to the code handling request parsing instead...